implement inverse transformation into input space

1-w opened the pull request:
automatically creates predictions transformed back to input t1 and flair space

simple solution that works with the "Affine" choice for registration.

Commit:
merged with inverse transformation into input space

1-w commented:
possible solution to #8

ravnoor commented:
This is great - thanks for contributing!

It looks like you used a code formatting tool introducing more changes than the intent of the commit. Would it be possible to separate these two changes (inverse transformation and code formatting) into two separate commits?

I should then be able to review the changes and merge the PR.

ravnoor committed:
revert inconsistent code formatting
- separate transformation functionality from code formatting

ravnoor committed:
rename image transform inputs
- maintain consistent naming scheme with the upstream NOEL-MNI/deepMask@d4c460a

ravnoor committed:
add `antspyx`

ravnoor committed:
specify if transform(s) should be inverted
- use forward or inverse transform

ravnoor committed:
create temp folder for intermediate outputs

ravnoor committed:
update `conda` version

ravnoor committed:
optimize layers for rebuilding

ravnoor committed:
remove redundant console outputs

ravnoor committed:
update NOEL-MNI/deepMask submodule
- write transforms to disk
- refactor requirements

ravnoor committed:
refactor and use intuitive variable names
- write transformed uncertainty image

ravnoor committed:
explicitly define affine

ravnoor committed:
print cleanup

ravnoor committed:
add missing argument

ravnoor committed:
add support for resampling [currently inactive]

ravnoor commented:
Reviewed the code, made the necessary changes, and tested the pipeline to ensure it runs as intended
